chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/certifiedProduct/CertifiedProductValidator.java

```diff
@@ -7,7 +7,8 @@
     static final long AMBULATORY_CQM_TYPE_ID = 1;
     static final long INPATIENT_CQM_TYPE_ID = 2;
     static final String URL_PATTERN = "^https?://([\\da-z\\.-]+)\\.([a-z\\.]{2,6})(:[0-9]+)?([\\/\\w \\.\\-\\,=&%#]*)*(\\?([\\/\\w \\.\\-\\,=&%#]*)*)?";
-
+    static final String EMAIL_PATTERN = "^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+.[A-Za-z]{2,}$";
```

Comment:
How sure are you this is the correct regex for email? Do we _have_ to have it? I'm leery of email validation via regex